QUARKUS-6964.md

```diff
@@ -0,0 +1,42 @@
+# QUARKUS-6964 - Add support for multiple cache types (backends)
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6964
+
+Add support for multiple cache types (backends)
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/50007
+
+Documentation:
+- https://quarkus.io/guides/cache#multiple-backends-simultaneously
+
+## Scope of testing
+This feature adding option to use multiple cache backend simultaneously.
+The tests will verify that it's possible set 2 cache providers (backends) at same time.
+The Caffeine and Redis providers will be used for testing.
+The tests coverage will be similar to already created tests for each provider. 
+The tests will cover both `@ApplicationScoped` and `@RequestScoped` to check if caching is done correctly.
+The testing will cover the ability to cache requests and also invalidate cache.
+
+## Automated test development
+For the testing of multiple backend providers at same time the new module with name `multi-provider` will be created.
+
+The automated tests will cover `@ApplicationScoped` and `@RequestScoped` for caching and invalidation requests.
+The caching will be tested for global cache name and for the named cache defined by `@CacheKey`.
```

Comment:
```suggestion
The caching will be tested for global cache name and for the named caches configured with `@CacheResult(cacheName=...)`.
```

Reply:
I understand this differently from the guide [cacheresult](https://quarkus.io/guides/cache#cacheresult).
The doc added in the upstream PR : https://github.com/elirenato/quarkus/blob/022b3f9062fff164ce79bf9cb5c32b4957e397f9/docs/src/main/asciidoc/cache.adoc?plain=1#L787 so relies on the Cache Name to decide if use one or other(redis or caffeine)...
`@CacheKey` is used only to generate keys inside the caches, what you say leads me to confusion about what is actually tested.
For that reason, I will put something like "The caching will be tested for the default cache and for named caches (via` @CacheResult(cacheName=...`)